### DISCUSSION TITLE
2,4kW bei 5% WR Limit

### DISCUSSION THREAD

**Discussion by keinprofi** [tbnobody/OpenDTU#2066] [Q&A]
### What happened?

Hi,

vorhin habe ich bemerkt, dass ich rund 2,4kW erzeuge, obwohl sämtliche WR auf 5% gedrosselt sind. Ein Bild sagt mehr als 1000 Worte:
![grafik](https://github.com/tbnobody/OpenDTU/assets/131543857/fe99bdd8-3f80-4030-8818-087af8b976e7)

Und ja, es sind wirklich alle 3 auf 5% gedrosselt (also 225 Watt max zusammen).

Jemand das schonmal so gehabt oder gesehen, bzw. eine Idee dazu? Möglicherweise ein bug im WR?

Ich habe die Leistung manuell verändert, auf 50%, 100%, 6%. Alles ohne sichtbaren Effekt. Mein Stromzähler hat die erzeugte Leistung bestätigt, ich hatte zeitweise rund 2kW eingespeist (bei eigentlich ZeroExport).

Beste Grüße
Karsten

### To Reproduce Bug

Sonne scheint auf die PV Module

### Expected Behavior

Ich habe erwartet, dass die WR Leistung bei 5% max 225 Watt sind (2x HMS 2000 und 1x HMS 500).

### Install Method

Pre-Compiled binary from GitHub

### What git-hash/version of OpenDTU?

v24.6.10

### Please confirm the following

- [X] I believe this issue is a bug that affects all users of OpenDTU, not something specific to my installation.
- [X] I have already searched for relevant existing issues and discussions before opening this report.
- [X] I have updated the title field above with a concise description.

**Comment by tbnobody** [comment]
Ich verschiebe das mal in die Q&A section da das hier für spezifische Issues und nicht für individuelle Fehler ist.

* Welche Leistung zeigen denn die Wechselrichter selbst an? 
* Steht die Prozent anzeige auch noch nach 15min auf 5%? (spät. nach dieser Zeit werden die aktuellen Werte wieder vom WR zurückgelesen)

**Comment by keinprofi** [comment]
Danke für's Verschieben.

Die Werte zeigen durchgehend 5% solange die Sonne ballert. Ich glaube aber ich weiß woran es liegen könnte. Eine Anpassung des Grid Profils durch Hoymiles könnte des Rätsels Lösung sein.
Die haben den Wert auf 0 angepasst:
![WhatsApp Bild 2024-06-15 um 12 24 42_3b456599](https://github.com/tbnobody/OpenDTU/assets/131543857/cf24bcce-e2ba-45aa-969d-9a10b77b9491)

Ich habe die openDTU gerade abgeschaltet, weil ich versuche per HoymilesDTU auf die WR zu kommen. Allerdings bekomme ich gerade keine Verbindung zum Hoymiles Cloud Server mit deren DTU.

**Comment by keinprofi** [comment]
Erste Ergebnisse:

Bei 2 WR habe ich das Grid Profil bereits anpassen können. das Limit schien noch drin zu sein, die Leistung (über FritzDECT ausgelesen) passt jedenfalls zu den 5%.
Mit dem 3. WR "kämpfe" noch damit er online ist in der HM Cloud...

Kurz zum Hintergrund. Hoymiles hatte das Grid Profil angepasst, da ein WR (genauer der500er) morgens nur auf 10 Watt kamm, trotz 100% Limit und SOnne. Erst nach einem Neustart lief der problemlos. Nach anpassen des Grid Profils lief der heute früh sauber an, aber war nicht mehr steuerbar. Und weil ich gleich 3 WR habe, haben die gleich alle 3 angepasst...

edit:
Sieht gut aus soweit, das scheint das Problem gewesen zu sein.
![grafik](https://github.com/tbnobody/OpenDTU/assets/131543857/94981286-20c9-464c-87c9-72d18a3e187f)


Also obacht wenn Hoymiles euer Grid Profil anpassen sollte weil etwas nicht läuft!!
